app/assets/javascripts/components/GlobalNav.js

```diff
@@ -304,20 +304,35 @@ define(['jquery', 'DoughBaseComponent', 'mediaQueries', 'utilities', 'common'],
   GlobalNav.prototype._toggleMobileNav = function() {
     this.$globalNav.toggleClass('is-active');
     this.$mobileNavOverlay.toggleClass('is-active');
+    $('body').addClass('no-scroll');
 
     // If we just closed the nav, reset all subnavs
     if (!this.$globalNav.hasClass('is-active')) {
       this.$globalNav.removeClass('is-active');
       this.$mobileNavOverlay.removeClass('is-active');
       this.$globalNavClumps.removeClass('is-active');
       this.$globalNavClump.removeClass('is-active');
+      $('body').removeClass('no-scroll');
     }
   };
 
   GlobalNav.prototype._toggleMobileSubNav = function(index) {
+    var siblingsNav = $(index).siblings().get(0);
+
+    $(index)
+      .parents('[data-dough-nav-clump]').toggleClass('is-active')
+      .parents('[data-dough-nav-clumps]').toggleClass('is-active');
+
+    $(siblingsNav).removeClass('is-hidden');
+  };
+
+  GlobalNav.prototype._returnMobileNav = function(index) {
     $(index)
       .parents('[data-dough-nav-clump]').toggleClass('is-active')
       .parents('[data-dough-nav-clumps]').toggleClass('is-active');
+      setTimeout(function(){
+        $('[data-dough-subnav]').addClass('is-hidden');
+      }, 400);
   };
```

Comment:
I wondered if this could be added in the CSS as a transition-delay property. The animation of the nav is done there so would make sense to keep it together and would share the variable for the time delay.

Reply:
Per our conversation: `transition-delay` can't be applied to the `display` property.